src/runtime/GlobalObject.cpp

```diff
@@ -40,6 +40,9 @@
 #include "EnvironmentRecord.h"
 #include "Environment.h"
 
+#include "ObjectTemplate.h"
+#include "FunctionTemplate.h"
+
```

Comment:
Why are these header files included? It seems that there is no usage in this cpp file.

Reply:
Yes. I was for test purpose. I removed it